pkg/dao/helpers_test.go

```diff
@@ -0,0 +1,34 @@
+package dao
+
+import (
+	"github.com/stretchr/testify/assert"
+)
+
+func (s *RepositorySuite) TestConvertSortByToSQL() {
+	t := s.T()
+
+	sortMap := map[string]string{
+		"name":          "name",
+		"url":           "url",
+		"package_count": "banana",
+		"status":        "status",
+	}
+
+	asc := ":asc"
+	desc := ":desc"
+
+	result := convertSortByToSQL("name", sortMap)
```

Comment:
looking at the function, am I right that it supports multiple sort parameters i.e. "name:asc,version:desc"? If so I think there should be a test for that

Reply:
could also test passing it an empty string

Reply:
That gave me a good idea about trimming whitespace..